### ISSUE TITLE
Wallet send to other account via self command error "not yet implemented"

### ISSUE THREAD

**Issue by Onefox** [mimblewimble/grin#2014]
I have 100 grin in my account and would like to send 3 to another account called test.
The grin build is from yesterdays master gdd8b4fa.
When trying to move coins from one account to another via the command
```sh
~$ grin wallet -a "default" send -m self -d "test" 3
```

It doesn't work and throws the following error:
```sh
not yet implemented
```

This is the log output in trace mode:
```sh
~$ grin wallet -a "default" send -m self -d "test" 3
20181123 14:59:55.112 INFO grin_util::logger - log4rs is initialized, file level: Debug, stdout level: Trace, min. level: Trace
20181123 14:59:55.113 INFO grin - Using wallet configuration file at /Users/xxx/.grin/grin-wallet.toml
20181123 14:59:55.113 INFO grin - This is Grin version 0.4.1 (git 0.4.0-127-gdd8b4fa), built for x86_64-apple-darwin by rustc 1.30.1 (1433507eb 2018-11-07).
20181123 14:59:55.113 DEBUG grin - Built with profile "release", features "".
20181123 14:59:55.116 INFO grin_wallet - Using LMDB Backend for wallet
20181123 14:59:55.116 DEBUG grin_wallet::types - Using wallet seed file at: /Users/xxx/.grin/wallet_data/wallet.seed
20181123 14:59:55.139 DEBUG grin_wallet::libwallet::internal::updater - Refreshing wallet outputs
20181123 14:59:55.151 DEBUG grin_wallet::libtx::build - Building input (spending regular output): 100000000000, 0300000000000000000000000100000000
20181123 14:59:55.151 DEBUG grin_wallet::libwallet::internal::selection - Building change outputs: total change: 96992000000 (1 outputs)
20181123 14:59:55.162 DEBUG grin_wallet::libtx::build - Building output: 96992000000, Commitment(0910bc4923859ec958452909b50f10ace41cba92b2d501f87944822c65c439e385)
20181123 14:59:55.226 INFO grin::cmd::wallet - Tx created: 3.000000000 grin to test (strategy 'all')
20181123 14:59:55.288 ERROR grin_util::logger -
thread 'main' panicked at 'not yet implemented': wallet/src/adapters/null.rs:38stack backtrace:
   0:        0x10869a32e - backtrace::backtrace::trace::h50ca359e02676fd7
   1:        0x10869b0fc - <backtrace::capture::Backtrace as core::default::Default>::default::h68c71254eb9042a6
   2:        0x10869b17d - backtrace::capture::Backtrace::new::h0a7e806c551310de
   3:        0x10861b731 - grin_util::logger::send_panic_to_log::{{closure}}::h202e0c64ebbab3c9
   4:        0x1087be608 - std::panicking::rust_panic_with_hook::h208eb16bfa9d451d
   5:        0x1081d4a97 - std::panicking::begin_panic::h8ed60d77fb7db96d
   6:        0x1081bc3bb - <grin_wallet::adapters::null::NullWalletCommAdapter as grin_wallet::adapters::WalletCommAdapter>::send_tx_sync::h1a4a95c1688ab2d9
   7:        0x10800ae3b - grin::cmd::wallet::wallet_command::{{closure}}::hb69579601cfc767e
   8:        0x107ff740b - grin_wallet::controller::owner_single_use::h09bf8a5f9e6c8717
   9:        0x107fd6e41 - grin::cmd::wallet::wallet_command::hb986dc1b29d997d4
  10:        0x10801fe73 - grin::real_main::hb82f6868531ada48
  11:        0x10801c558 - grin::main::h4592fc029c566f10
  12:        0x107f87205 - std::rt::lang_start::{{closure}}::h4675e75bcff76258
  13:        0x1087bdf87 - std::panicking::try::do_call::h682cd0c950e47076
  14:        0x1087cbe3e - ___rust_maybe_catch_panic
  15:        0x1087b37fc - std::rt::lang_start_internal::heabafbf362cc3e1e
  16:        0x10802137b - _main
 
 
Thread 'main' panicked with message:
"not yet implemented"
```

**Comment by yeastplume** [comment]
fixed via #2022.. please test away and feel free to reopen if this is still an issue

**Comment by Onefox** [comment]
Works now thanks!
